classes/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_pamission: true)
+    @name = name
+    @age = age
+    @parent_pamission = parent_pamission
+    @id = Time.new.to_f
+  end
+
+  attr_reader(:id)
+  attr_accessor(:name, :age)
+
+  def can_use_services?
+    (@parent_pamission && of_age?) || false
+  end
```

Comment:
- [ ] Ouch! this is a close one, but the `can_use_services?` method should return true if the person is of age or if they have permission from parents, so I'd advise you change the `&&` to an `||`.